### DISCUSSION TITLE
Just FYI, the Dirt Rally profile also works right out of the Box for Grid Legends (and probably F1 too)

### DISCUSSION THREAD

**Discussion by jansteffen** [cosmii02/RacingDSX#46] [Show and tell]
Just as the title says, RacingDSX works with Grid Legends using the Dirt Rally Profile with no issues. It requires the same config tweak to `hardware_settings_config`, but no other adjustments are needed. That being said, because the game's handling is so much more arcady and forgiving when it comes to traction you're not getting a huge amount of feedback, but it's still cool.

I haven't had a chance to try it with any of the F1 games yet, but given that they're all on the same engine and probably share a huge amount of tech I wouldn't be surprised if they also work out with the same setup.

**Comment by cosmii02** [comment]
Would be awesome if people would share their settings for non dirt or forza games
